DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -128,6 +128,10 @@ h1#page-heading {
 .responsive-iframe {
   width: 100%;
   height: 100%;
+
+  &.responsive-iframe-min-height {
+    min-height: 550px;
+  }

```

Comment:
Is there a reason we need this here? If we're using the rest of these iframe classes we already have a min height?

Reply:
Yes, despite the wrapper having the min-height, in all the browsers I tested the Iframe would not stretch to fill that space and would only fill around half of that.
I tried in a Chrome, Firefox, and Edge - they all appeared to behave the same.
Weighing it up, is it worth spending time trying to figure this out or just add this extra bit of re-usable css?
Could you run this locally and see if you can re-create?

Reply:
Your new support tickets page doesn't behave the same as the Pricing (and others) pages for me in Chrome at least:
![image](https://user-images.githubusercontent.com/59561751/144022106-ee337045-1114-496f-9808-085a25bfa990.png)

But I also see what you mean about when this isn't there? (removed the new class from the wrapper div)
![image](https://user-images.githubusercontent.com/59561751/144022006-ab23be44-076a-436b-9bca-edfa8936cf5a.png)

I'm pretty sure the issue without the class is because we're stuck in a 2 column layout so it's only allowed to reach the bottom of the left column rather than the page. Ideally we would have the same behaviour as the other pages, where the column is allowed to the bottom of the page, but if that proves too difficult, I think what we have now is acceptable. Maybe time box for an hour to see if you can force the column full height without destroying the layout and if not we just go with what we have?

Reply:
Attempting a few things doesn't seems to work. 
If I remove the left column and basically match the pricing markup then it works as expected.
So there is something about the 2 column layout that is causing the issue.

Reply:
I guess we'll have to leave it for now then, I wouldn't be surprised if this comes back in test though.

Reply:
I think it might come back (possibly in UAT) as a "Why do we have to scroll this small box when there is all the space on the page"

Reply:
Ah ok I see. I will merge this in and we can then re-think if this end result is not acceptable.